email_alert_service/models/email_alert_template.rb

```diff
@@ -20,7 +25,9 @@ def message_body
 private
 
   def make_url_from_document_base_path
-    Plek.new.website_root + @document["base_path"]
+    base_path = @document["base_path"]
+    content_item = @content_store.content_item(base_path)
+    link = content_item.links.parent[0].web_url
```

Comment:
Are we very sure documents here will always contain `parent.links`?

It may be better to use hash-syntax here: `content_item['links']['parent'].first['web_url']` (api-adapters returns allows for method-style access via a OStruct, but we might remove that in the future as its causing a lot of problems).